docker-compose.yaml

```
@@ -86,18 +80,19 @@ services:
     #  - ./server.crt:/usr/local/apache2/conf/server.crt
     #  - ./server.key:/usr/local/apache2/conf/server.key
 
+services:
+  crs-apache:
+    image: owasp/modsecurity-crs:apache
+    ports:
+      - "80:80"
+      # only available if SETTLS was enabled:      
+      - "443:443"
+    <<: *default-settings
+
   crs-nginx:
-    extends: crs-apache
     image: owasp/modsecurity-crs:nginx
     ports:
-      - "80:8080"
+      - "80:80"
```

That port mapping is intentional ;)
```suggestion
      - "80:8080"
```